### ISSUE TITLE
[19w28a][Suggestion] Rail length limitation?

### ISSUE THREAD

**Issue by Tommyg930** [tsunamayo/Skywanderers#1376]
TL:DR at the end

I love the new look for the rails, it makes way more sense to have them be a block rather then a surface feature.
Currently I'm working on a Capital Industrial ship. this ship is made to have 3 huge sliding doors, with one door spanning 25 blocks. the first one will need to move back 26 blocks, the second will need to move back 51 blocks and the last one will need to move black 76 blocks.

Obviously this is currently an issue as the longest I can make a rail is 16 blocks. and while I can stack the rail blocks, this causes issues as the first rail block will move slower then the one on top of it, and it will take way more room to accommodate all the rails-on rails-on rails. 
I remember having rail extender blocks a few versions back, so are they going to be reintroduced any time soon? 

Work in progress view of the hanger gap where the 3 doors are supposed to go. Corillian sitting in the doorway for scale.
![20190617053734_1](https://user-images.githubusercontent.com/33021337/61504912-65112600-a9dd-11e9-81fa-10c4e5241630.jpg)

TL;DR: Plz bring back rail extenders

**Comment by tsunamayo** [comment]
Yes I removed them because it felt like a nightmare to code with the new tech (and I want to stay far away from problematic like these). But I got idea on how to solve that elegantly, using the new design to my advantage. You would have to put the extension exactly like an addon block (which they would be essentially). I keep you posted.

**Comment by Tommyg930** [comment]
That sounds great to me, looking forward to it.

**Comment by Xerma** [comment]
I too desperately need this as my 236m diameter sphere ship, based on the Core Ship, is going to have quite a few levels. Looking forward to seeing what you come up with :-]

**Comment by tsunamayo** [comment]
I added new extension block for both rails, using the new addon tech like the capacitor. Will be in 19w31a (hopefully tonight). Cheers!
